### PR TITLE
video thumbnail & Top level page lagging  UI fixes

### PR DESCRIPTION
**Pull request by rakeshAlgo**
Hi @amarantha-k ,
I have fixed video thumbnail & small lagging issues on the top level landing page.

### CODE REVIEW

File: src/js/01-nav.js [hunk 1]
```diff
@@ -133,7 +133,7 @@
   }
 
   // hide left nav on top level landing page
```
Comment (mojavelinux) [inline src/js/01-nav.js:135]: It should not be necessary to use JavaScript to hide the nav on the top level landing page. You can simply remove it from the template (and adjust the CSS if necessary).

**Review by mojavelinux — CHANGES_REQUESTED** [review]

**Comment by mojavelinux** [comment]
@amarantha-k The deploy previews are working again.

File: src/js/01-nav.js (hunk 1 above)
Comment (amarantha-k) [reply src/js/01-nav.js:135]: @rakeshAlgo please update this PR with the (new) landing page template for top-level page when it's available.

Comment (rakeshAlgo) [reply src/js/01-nav.js:135]: Hi @amarantha-k ,
Please merge this PR, in the next PR I"ll remove hide nav script for top-level page. 
And will create a new landing page without the navigation for the top-level landing page.

**Review by amarantha-k — APPROVED** [review]
Merging to pick up UI updates. The update to remove script in landing pages will be submitted in the next PR.

**Pull request merged**